src/interceptors/fetch/index.ts

```diff
@@ -26,9 +26,9 @@ export class FetchInterceptor extends Interceptor<HttpRequestEventMap> {
   }
```

Comment:
Afaik, `global.fetch` is synonymous to `window.fetch`. `global` is `window` in a browser environment. So I think it should be safe to perform the check on `global` instead of `window` 👍

Reply:
With the only correction: `globalThis`* 

`global` is a Node.js global variable. We should use `globalThis`—that one is synonymous to `window` in a browser and to `global` on Node.js.